pt-BR/active_record_querying.md

```diff
@@ -1546,14 +1546,14 @@ Client.unscoped {
 }
 ```
 
-Dynamic Finders
+Localizadores Dinâmicos
 ---------------
 
-For every field (also known as an attribute) you define in your table, Active Record provides a finder method. If you have a field called `first_name` on your `Client` model for example, you get `find_by_first_name` for free from Active Record. If you have a `locked` field on the `Client` model, you also get `find_by_locked` method.
+Para cada campo (também conhecido como atributo) que você define na sua tabela, o Active Record fornece um método localizador. Se você tiver um campo chamado `first_name` no seu *model* `Client` por exemplo, você terá de graça o método `find_by_first_name` fornecido pelo Active Record. Se você tiver o campo `locked` no seu *model* `Client`, você também receberá o método `find_by_locked`.
```

Comment:
```suggestion
Para cada campo (também conhecido como atributo) que você define na sua tabela, o *Active Record* fornece um método localizador. Se você tiver um campo chamado `first_name` no seu *model* `Client` por exemplo, você terá de graça o método `find_by_first_name` fornecido pelo *Active Record*. Se você tiver o campo `locked` no seu *model* `Client`, você também receberá o método `find_by_locked`.
```

Reply:
Ops, pequeno deslize em não colocar o Active Record como itálico :sweat_smile: